packages/core/src/Camera.ts

```diff
@@ -17,9 +16,16 @@ import { Transform } from "./Transform";
 import { UpdateFlag } from "./UpdateFlag";
 
 /**
- * @todo
+ * Camera clear flags enumeration.
  */
-type Sky = {};
+export enum CameraClearFlags {
```

Comment:
Use seperate file , for example `*/enums/CameraClearFlags.ts` path.

Reply:
+1